Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -409,10 +409,13 @@ private func expandBodyMacro(
     return nil
   }
 
-  // Wrap the body in braces.
-  let beforeBody = decl.body == nil ? " " : "";
-  let indentedSource = beforeBody + "{\n" + expanded.indented(by: decl.indentationOfFirstLine + indentationWidth) + "\n}\n"
-  return "\(raw: indentedSource)" as CodeBlockSyntax
+  // `expandAttachedMacro` adds the `{` and `}` to wrap the accessor block and
+  // then indents it.
+  // Remove any indentaiton from the first line using `drop(while:)` and then
+  // prepend a space to separate it from the variable declaration
```

Comment:
This comment seems out of date, it references accessor blocks and variables.

Reply:
Ah, thanks. It also has a copy-pasted type (indentaiton). I'll fix both

Reply:
Haha, it seemed like the sort of typo I made.

Reply:
The comment is still incorrect, though. It mentioned accessor block and variables but this is about function bodies.

Reply:
Yup, follow-up PR is at https://github.com/apple/swift-syntax/pull/2392